### ISSUE TITLE
Mocaccino skel missing /var/lock directory

### ISSUE THREAD

**Issue by jcfrosty** [mocaccinoOS/desktop#151]
/var/lock folder is missing which breaks luet functionality. 
Not sure if luet should be more aware and capable of creating the missing directory or if we just need to make sure skel creates the directory or both.

**Comment by geaaru** [comment]
IMHO, i think that it's something to fix in luet. Atm, it does the same for the TMPDIR directory IIRC.

**Comment by mudler** [comment]
> IMHO, i think that it's something to fix in luet. Atm, it does the same for the TMPDIR directory IIRC.
> 
> Hereinafter, what is been done for tmpdir. We need something similar for /var/lock:
> https://github.com/mudler/luet/blob/master/pkg/config/config.go#L407

I totally disagree there. It is out of purpose to create system directories which should be present. You can disable locking with `LUET_NOLOCK`, but by default you should have a proper filesystem layout.

This is also to avoid imposing any default permission bits to the filesystem structure, that it should be instead present already.

**Comment by joostruis** [comment]
This commit solves this issue too?
https://github.com/mocaccinoOS/os-commons/commit/8b452209210edb9f6e883525fd9760bfba90c175
